crates/fluvio-cli/src/install/plugins.rs

```diff
@@ -28,50 +34,100 @@ pub struct InstallOpt {
     /// If the package ID contains a version (e.g. `fluvio/fluvio:0.6.0`), this is ignored
     #[clap(long)]
     pub develop: bool,
+
+    /// When this flag is provided, use the hub. Dev-only
+    #[clap(long, hide_short_help = true)]
+    pub hub: bool,
+
+    /// When this flag is provided, use the hub. Dev-only
+    #[clap(long, hide_short_help = true)]
+    pub channel: Option<String>,
+
+    /// When this flag is provided, use the hub. Dev-only
+    #[clap(long, hide_short_help = true)]
+    pub target: Option<String>,
 }
 
 impl InstallOpt {
     pub async fn process(self) -> Result<()> {
-        let agent = match &self.prefix {
-            Some(prefix) => HttpAgent::with_prefix(prefix)?,
-            None => HttpAgent::default(),
-        };
+        if self.hub {
+            debug!("Using the hub to install");
 
-        // Before any "install" type command, check if the CLI needs updating.
-        // This may be the case if the index schema has updated.
-        let require_update = check_update_required(&agent).await?;
-        if require_update {
-            prompt_required_update(&agent).await?;
-            return Ok(());
-        }
+            let mut homedir_path = fluvio_bin_dir()?;
+            let package_name;
+            //let binpath = "sample-bin-script";
+            let bin_install_path = if let Some(ref p) = self.package {
+                let package = p;
+                package_name = package.name().to_string();
+                homedir_path.push(package_name.clone());
+                homedir_path.to_str().ok_or(crate::CliError::Other(
+                    "Unable to render path to fluvio bin dir".to_string(),
+                ))?
+            } else {
+                return Err(crate::CliError::Other("No package name provided".to_string()).into());
+            };
+            debug!(?bin_install_path, "Install path");
 
-        match self.install_plugin(&agent).await {
-            Ok(_) => (),
-            Err(err) => match err.downcast_ref::<CliError>() {
-                Some(crate::CliError::IndexError(fluvio_index::Error::MissingTarget(target))) => {
-                    install_println(format!(
-                        "❕ Package '{}' is not available for target {}, skipping",
-                        self.package.name(),
-                        target
-                    ));
-                    install_println("❕ Consider filing an issue to add support for this platform using the link below! 👇");
-                    install_println(format!(
-                            "❕   https://github.com/infinyon/fluvio/issues/new?title=Support+fluvio-cloud+on+target+{}",
+            let access =
+                HubAccess::default_load(&Some("https://hub-dev.infinyon.cloud".to_string()))
```

Comment:
If this line does a `default_load(None)` then the regular set of hub remote defaults and overrides will apply allowing prod, or dev access with some config.

Reply:
Now there are 2 ways to opt-out of using dev:
1. If env vars `INFINYON_HUB_REMOTE` or `FLUVIO_HUB_PROFILE` are set
2. If `--use-hub-defaults` is used on CLI

Like this
```shell
fluvio install sample-bin-script --hub --use-hub-defaults
```

(When we are ready to make this the default install path, we won't need this CLI flag anymore)